### PR TITLE
[release-1.14] remove extra ef

### PR DESCRIPTION
**Pull request by hanxiaop**
**Please provide a description of this PR:**
1.11 and 1.15 should not be included.

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [x] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml [hunk 1]
```diff
@@ -2677,248 +2435,6 @@ spec:
                     local:
                       inline_string: "envoy.wasm.stats"
 ---
-# Source: istiod/templates/telemetryv2_1.15.yaml
```
Comment (zirain) [inline manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml:2680]: we always add next release in prev releases.

**Comment by ericvn** [comment]
I'm not sure if .12 gets removed since it's EOL.

File: manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml (hunk 1 above)
Comment (hanxiaop) [reply manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml:2680]: @zirain I just looked at other release branches, and they only contain the corresponding release and two prev releases. Not sure if 1.15 should be included in the 1.14 release.

**Comment by ericvn** [comment]
> @ericvn I think for 1.14, it should be included?

That's fine. Maybe we only remove in minor releases (and thus its patches) after it's EOL.

**Pull request closed without merging**